### DISCUSSION TITLE
Please Add Multi-Tenancy support and features!

### DISCUSSION THREAD

**Discussion by sundialrama** [FlowiseAI/Flowise#1180] [Ideas]
Hey please add the ability for multiple users to login and use flowise at the same time!

**Comment by gitgrooves** [comment]
yes please!!!

**Comment by materod-upv** [comment]
Yes please, I need to have multiple users with different flows.

**Comment by Learning420** [comment]
i am working on something I have it almost configured will shaare when i get it working better.
